pages/_slug.vue

```diff
@@ -0,0 +1,84 @@
+<template>
+  <section-page
+    v-if="page"
+    :name="slug"
+    :descendant-of="page.id"
+  />
+</template>
+
+<script>
+import gtm from '~/mixins/gtm'
+
+export default {
+  name: 'Section', // this is the template name which is used for GTM
+  components: {
+    SectionPage: () => import('../components/SectionPage')
+  },
+  mixins: [gtm],
+  async asyncData ({
+    $axios,
+    params,
+    error
+  }) {
+    const path = `${params.slug}`
+    const page = await $axios.get(`/pages/find/?html_path=${path}`)
+      .catch(() => {
+        error({
+          statusCode: 404,
+          message: 'Page not found'
+        })
+      })
+    return {
+      page: page?.data
+    }
+  },
+  data () {
+    return {
+      page: null,
+      slug: this.$route.params.slug
+    }
+  },
+  head () {
+    return {
+      title: this.slug + ' - Gothamist',
+      meta: [
+        {
+          hid: 'description',
+          name: 'description',
+          content: this.slug + ' - Gothamist'
+        },
+        {
+          hid: 'og_url',
+          name: 'og:url',
+          content: 'https://www.gothamist.com' + this.$route.fullPath
+        },
+        {
+          hid: 'og_title',
+          name: 'og:title',
+          content: this.slug + ' - Gothamist'
+        },
+        {
+          hid: 'og_description',
+          name: 'og:description',
+          content: this.slug + ' - Gothamist'
+        },
+        {
+          hid: 'twitter_title',
+          name: 'twitter:title',
+          content: this.slug + ' - Gothamist'
+        },
+        {
+          hid: 'twitter_description',
+          name: 'twitter:description',
+          content: this.slug + ' - Gothamist'
+        },
+        {
+          hid: 'twitter_url',
+          name: 'twitter:url',
+          content: 'https://www.gothamist.com' + this.$route.fullPath
```

Comment:
Might be worth confirming with some test on demo, but it looks like we don't actually need twitter properties that are redundant with og data.

https://developer.twitter.com/en/docs/twitter-for-websites/cards/guides/getting-started#opengraph
> When using Open Graph protocol to describe data on a page, it is easy to generate a Twitter card without duplicating tags and data. When the Twitter card processor looks for tags on a page, it first checks for the Twitter-specific property, and if not present, falls back to the supported Open Graph property. This allows for both to be defined on the page independently, and minimizes the amount of duplicate markup required to describe content and experience.

Reply:
hmm ok I'll remove so we can test this out